### PR TITLE
Update ftc-at-ethcc.md

### PR DESCRIPTION
**Pull request by laeccles**
Update to FTC @ EthCC

### CODE REVIEW

File: content/events/ftc-at-ethcc.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+venueName: "Sorbonne University"
+venueAddress: "15-21 Rue de l'École de Médecine"
+venueMapsLink: "https://goo.gl/maps/d2fYuBMNrYmi7sQo7"
+website: "https://fundingthecommons.io/paris-2023"
 startDate: 07/15/2023
 endDate: 07/16/2023
 tag: PLN Event
 dateTBD: false
+preferredContacts:
+- 'Twitter|https://twitter.com/fundingcommons'
```
Comment (WinsCodes) [inline content/events/ftc-at-ethcc.md:15]: @laeccles Please follow the same spacing format just like the eventHosts attribute - 

preferredContacts:
  - 'Twitter|https://twitter.com/fundingcommons'

File: content/events/ftc-at-ethcc.md [hunk 2]
```diff
@@ -1,11 +1,18 @@
 ---
 eventName: FTC at EthCC
+eventDescription: "Funding the Commons is focused on new models of sustainable public goods funding and value alignment in open source networks. We bridge the public goods community across Web2, Web3, research, philanthropy and industry. We do this by convening builders, academics, and funders in both traditional conference and "open space" formats."
```
Comment (WinsCodes) [inline content/events/ftc-at-ethcc.md:3]: @laeccles Please update the content in the description's last line - if open space is required to be enclosed within double quotations, use the escape character \ before the double quotations just like below - 
\"open space\"

File: content/events/ftc-at-ethcc.md (hunk 1 above)
Comment (WinsCodes) [inline content/events/ftc-at-ethcc.md:15]: @laeccles Please update 'Twitter' to 'twitter'

Comment (laeccles) [reply content/events/ftc-at-ethcc.md]: @WinsCodes I have updated this

File: content/events/ftc-at-ethcc.md (hunk 2 above)
Comment (laeccles) [reply content/events/ftc-at-ethcc.md]: I have updated this

**Comment by WinsCodes** [comment]
The changes have been merged using the Pull request: https://github.com/memser-spaceport/pln-events/pull/351.

Closing this Pull request.

**Pull request closed without merging**